utils/download-references.sh

```diff
@@ -5,31 +5,31 @@ if [ $# -eq 0 ]; then
 else
     DEST=$1
     if [ -d "$DEST" ]; then
-        mkdir $DEST/rnafusion_references
-        cd $DEST/rnafusion_references
-
+        cwd=$DEST/rnafusion_references
+        mkdir $cwd
```

Comment:
Can you make it a one-liner?
`mkdir $cwd && cd $cwd`

Reply:
done